scripts/single-node.sh

```diff
@@ -103,6 +106,7 @@ startCelestiaApp() {
     --grpc.enable \
     --grpc-web.enable \
     --timeout-commit 1s \
+    --rpc.grpc_laddr tcp://0.0.0.0:9098 \
```

Comment:
grpc is no longer 9090?

Reply:
GRPC is still 9090 but there is another service running on 9098 that apparently just supports `/broadcast_tx_commit`

Reply:
The service being exposed on 9098 here is the cometbft grpc api. It exposes both broadcast api and more importantly, the block api introduced by celestia-core and consumed by celestia-node. 

It only truly needs to be turned on when running celestia-app through the multiplexer in standalone mode - as the app runs out of process to the celestia-core/multiplexer. In this case the app runs a proxy service to connect to the block api. There is a thread with full context in v4-protocol channel, if you have any other questions I can try answer them